test/pom.xml

```diff
@@ -245,6 +245,12 @@ THE SOFTWARE.
           </dependency>
         </dependencies>
       </plugin>
+      <plugin>
+        <artifactId>maven-deploy-plugin</artifactId>
```

Comment:
See above. Related to the extent that Javadoc generation is triggered by running deployment in the release profile. Could be separated if you care.